course-v2/layouts/partials/course_description.html

```diff
@@ -9,7 +9,7 @@ <h4 class="course-detail-title">
   </h4>
   {{ if $shouldCollapseDescription }}
   <div id="collapsed-description" class="description">
-    {{- .context.RenderString (truncate 320 $courseData.course_description) -}}
+    {{- truncate 320 (.context.RenderString $courseData.course_description) -}}
```

Comment:
Truncating description before resolving shortcodes breaks the shortcodes. The order has been changes here to first render content (i.e. shortcodes) then truncate the output.

Reply:
It seems this was my fault in https://github.com/mitodl/ocw-hugo-themes/pull/876. @pdpinch I was able to confirm that this will fix https://github.com/mitodl/hq/issues/3858.